package.json

```diff
@@ -57,7 +57,7 @@
     "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-react": "^7.9.1",
     "npm": "^6.14.4",
-    "react-docgen": "^2.21.0",
+    "react-docgen": "^5.3.0",
```

Comment:
Wonder if a `~` would be better than `^` here (I just learned the differences [here](https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json))